### ISSUE TITLE
Problems with the String Quartets Volumes (Viola, especially)

### ISSUE THREAD

**Issue by NinoTerraza** [musescore/MuseScore#17641]
### Your idea

I've been trying for a while to make a Viola (Solo) increase its volume, using Forte, Fortissimo, etc. The increase is minimal, and it really exhausts being stuck in something that was so simple and easy before...
Other problems with Quartets:
1) Volume of Violin1 is higher (per se) than that of Violin2. It is necessary? I had to turn up the volume of the Violin2 in the mixer but still the inefficiency of the sound persists. I thought choosing Violin1 on that track to repeat it would sound good, however, apparently it's the track that has settings for volume and reverb effects.
2) Violin2, like the Viola, "suffers" a reverberation that is too flooding, which makes them lose the sense of SOLO. Can that be modified? As?
3) Cello I also had to force its volume...
Thank you very much for the help.

### Problem to be solved

Improve the functioning of the dynamics; fix volumes assigned to dynamics by default.

[Ejemplo Viola muy suave.zip](https://github.com/musescore/MuseScore/files/11512765/Ejemplo.Viola.muy.suave.zip)

**Comment by DaddyLudwig** [comment]
I've noticed the lack of power in Violin 2 Solo as well, this is why I can't switch the Violin 1 staff over to Violin 2 Solo in a Beethoven string quartet despite the staccato triggered grace note effect in Violin 1 Solo is cause I'd lose the power that Beethoven needs. I've only noticed a balance issue with cello in piano trios, not string quartets, and it's gotten a bit better since the previous update to strings. And viola, well, in the music I've been transcribing so far anyway, it's a pretty unnoticeable part compared to the violins and cello, even in professional string quartet recordings, so I haven't noticed an issue in the viola.

**Comment by NinoTerraza** [comment]
I usually compose for cuareda quartets where I give important parts of the work to each instrument.
I hope you see and hear the example that I uploaded as a file, so that you notice the real problem with the Viola, and that it is not expected in any way, much less for professional music.
Listening and commenting on me, after.

**Comment by DaddyLudwig** [comment]
I do notice it in your example. But in Beethoven or Mozart or whatever, I don't notice it, only the Violin 2 thing. And it's in those composers where the viola is sometimes unnoticeable in professional recordings. And it's those composers that I've been transcribing.

**Comment by NinoTerraza** [comment]
But did you notice that Viola was FFF versus Violins MF? That's not normal, sorry.

**Comment by DaddyLudwig** [comment]
Yes, I saw the fff.

**Comment by cbjeukendrup** [comment]
I assume all of this is about Muse Sounds rather than MS Basic, right?

**Comment by NinoTerraza** [comment]
Obviously, there are no other default libraries in MuseScore with Strings "Solo".

**Comment by bbalberc42** [comment]
I've noticed this as well, I've had to increase the volume level of the viola in the mixer every time to at least +6dB to even hear it w/ the rest of the instruments. The problem gets even worse when the viola is playing pizzicato, where it's barely audible at all, unless I increase its volume to +12dB or above.

When I uploaded the piece to MuseScore.com, of course the settings were persistent, so then the viola was overwhelmingly loud in the mix on the website itself. A fix for this would be much appreciated.

**Comment by MarcSabatella** [comment]
@bbalberc42 The audio on musescore.com should be identical to what you hear in MuseScore, assuming you use MuseScore to upload it (eg, with File / Publish to musescore.com) rather trying to use your browser for that.  If you have questions about that process, please ask for help on the Support forum on musescore.org

**Comment by Valdessir** [comment]
> Obviously, there are no other default libraries in MuseScore with Strings "Solo".

MS Basic also has solo string sounds.

**Comment by dr-Fade** [comment]
I had the same issue but with the latest MuseScore (4.4.4-243461245) and the latest Muse Strings (0.4.23) the issue seems to be gone for the solo versions of Viola and Cello.
The solo Violin 2 is still a bit quieter than Violin 1, tho.
